hax/hax/motr/hax.c

```diff
@@ -256,6 +256,14 @@ PyObject *m0_ha_proc_counters_fetch(unsigned long long ctx,
 		py_fid,
 		list);
 
+	
+	/* Allocation of pc_bckey and pc_bcrec happens in
+	* m0_filesystem_stats_fetch in motr code */
+	for (i=0; i < len; ++i) {
+		m0_free(count_stats.pc_bckey[i]);
```

Comment:
Ideally if hax does not allocate them explicitly, hax must not free them either. Either Motr must provide an api to free them or let user of m0_filesystem_stats_fetch() allocate and free them. I would prefer the later.